tensorflow/compiler/mlir/tosa/transforms/legalize_common.cc

```diff
@@ -1438,70 +1439,97 @@ llvm::Optional<Value> convertSoftmaxOp(PatternRewriter& rewriter, Operation* op,
       // Input is 9.7, where lower 7 bits are all zeros.
       // Output is 23 bits, where lower 7 bits should be all zeros as well,
       // since there's no interpolation here.
-      auto op5_table_op4_upper = CreateOpAndInfer<tosa::TableOp>(
+      auto op5_table_op4_01 = CreateOpAndInfer<tosa::TableOp>(
           rewriter, op->getLoc(), int32_logits_type, op4_rescale_op3,
-          exp_table_const_upper);
+          exp_table_const_01);
 
-      auto op6_table_op4_lower = CreateOpAndInfer<tosa::TableOp>(
+      auto op6_table_op4_02 = CreateOpAndInfer<tosa::TableOp>(
           rewriter, op->getLoc(), int32_logits_type, op4_rescale_op3,
-          exp_table_const_lower);
+          exp_table_const_02);
+
+      auto op7_table_op4_03 = CreateOpAndInfer<tosa::TableOp>(
```

Comment:
Hmm..Let me think

Reply:
How about `op7_table_op4_first8`,  `op7_table_op4_second8`, etc. something like those?

Reply:
The opN prefix means small changes end up with a lot of renaming, which I hope we can avoid in future. I don't know if we could have something like "first_order_lookup_table" or "newton_raphson_bits8t16" or something. I just don't get an intuition of what op7_table_op4_first8 is (the first8 is an improvement, table is useful too even if more what it is than what it represents).

Reply:
I totally agree with that and it's quite tedious when updating those opN. I am currently using the naming like `op8_table_op4_bits_7_0` and `op7_table_op4_bits_15_8`, etc. For all the opN names changes here, I believe it could be another PR which is solely for refactoring this to avoid the confusion between the original intention of this PR? What do you think? : )

Reply:
SGTM